airflow/providers/jenkins/operators/jenkins_job_trigger.py

```diff
@@ -168,6 +177,7 @@ def poll_job_in_queue(self, location: str, jenkins_server: Jenkins) -> int:
                     return build_number
             try_count += 1
             time.sleep(self.sleep_time)
+
         raise AirflowException(
             "The job hasn't been executed after polling " f"the queue {self.max_try_before_job_appears} times"
```

Comment:
While we’re changing this file, can you also fix this line too? It should be

```python
f"The job hasn't been executed after polling the queue {self.max_try_before_job_appears} times"
```

Reply:
applied